packages/manager/src/features/CloudPulse/Alerts/AlertsDetail/AlertDetail.tsx

```diff
@@ -117,6 +118,12 @@ export const AlertDetail = () => {
             <AlertDetailCriteria alert={alertDetails} />
           </Box>
         </Box>
+        <Box sx={getAlertBoxStyles(theme)}>
```

Comment:
```suggestion
        <Box sx={getAlertBoxStyles}>
```
theme will be automatically passed to function by sx props